### ISSUE TITLE
:gen-class is stripped

### ISSUE THREAD

**Issue by jsab** [gfredericks/how-to-ns#2]
## Expected behavior

`(:gen-class)` should be the last form in the namespace declaration.

## Actual behavior

`(:gen-class)` gets stripped.

````
Bad ns format:
--- a/src/test_record/core.clj
+++ b/src/test_record/core.clj
@@ -1,3 +1,2 @@
 (ns test-record.core
-  "My beautiful namespace."
-  (:gen-class))
+  "My beautiful namespace.")
````

## Steps to reproduce the problem

Call `lein how-to-ns check` on any namespace with a `(:gen-class)`.

**Comment by gfredericks** [comment]
this should be pretty easy -- we can print the whole thing on one line, and that's at least better than the current behavior.

**Comment by gfredericks** [comment]
Released with `0.1.4`.
